charts/lnd/templates/export-secrets-configmap.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ printf "%s-export-secrets" (include "lnd.fullname" .) }}
+  labels:
+    {{- include "lnd.labels" . | nindent 4 }}
+data:
+  exportSecrets.sh: |
+    #!/bin/sh
+    kubectl wait --for=condition=ready --timeout=150s pod {{ include "lnd.fullname" . }}-0
```

Comment:
could it take longer for the pod to start?
I guess if bitcoind is not ready we can still create the wallet?

Reply:
> could it take longer for the pod to start?
> I guess if bitcoind is not ready we can still create the wallet?

Yep, wallet can be created before lnd tries to talk to bitcoind, but I'm not sure if lnd will become ready until it can talk to bitcoind. But if a user installs bitcoind, he will have to wait for a long time before installing lnd due to the time it takes for bitcoind to sync. So I guess we cannot really take the bitcoind delay into account here and increase the timeout because it could be days.

Reply:
umbrel has a nice trick: [start lnd with neutrino, and switch when bitcoin is synced](https://github.com/getumbrel/umbrel/blob/79c011ea8ec9a97f3e68654f8d91dae048cc7b5b/scripts/configure#L209).

but I don't think we need this for what we do. we're targetting more large scale opeation, not a raspbery self usage solution.